tests/test_util/aiohttp.py

```diff
@@ -57,7 +58,8 @@ def call_count(self):
         return len(self.mock_calls)
 
     @asyncio.coroutine
-    def match_request(self, method, url, *, auth=None):
+    def match_request(self, method, url, *, auth=None, timeout=None): \
```

Comment:
Will remove. It is only applicable for long streaming type requests I guess, for most of our uses the default should be ok